rgd/main.go

```diff
@@ -0,0 +1,90 @@
+package main
+
+import (
+	"context"
+	"fmt"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"github.com/spf13/cobra"
+	"k8s.io/client-go/tools/clientcmd"
+
+	rgdv1alpha1 "github.com/aws/resource-graph-definition-controller/api/v1alpha1"
+	rgdclient "github.com/aws/resource-graph-definition-controller/pkg/client"
+)
+
+type ResourceNode struct {
+	ID           string
+	Status       string
+	Dependencies []string
+	Children     []*ResourceNode
+}
```

Comment:
`ResourceNode` is declared twice in this package. this program doesn't not compile.